cves/CVE-2016-4430.yml

```diff
@@ -6,33 +6,33 @@ nickname_instructions: |
   Nickname is optional. Provide a useful, professional, and catchy nickname for
   this vulnerability. Ideally fewer than 30 characters. This will be shown
   alongside its CVE to make it more easily distinguished from the rest.
-nickname:
+nickname: Better URL Validation
 
 CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE:
+CWE: CWE-352
 
 curated_instructions: |
   If you are manually editing this file, then you are "curating" it. Set the
   entry below to "true" as soon as you start. This will enable additional
   integrity checks on this file to make sure you fill everything out properly.
   If you are a student, we cannot accept your work as finished unless curated is
   set to true.
-curated: false
+curated: true
 
 reported_instructions: |
   Was there a date that this vulnerability was reported to the team? You can
   find this in changelogs, blogs, bug reports, or perhaps the CVE data.
   Please enter your date in YYYY-MM-DD format.
-reported:
+reported: 2016-06-11
 
 announced_instructions: |
   Was there a date that this vulnerability was announced to the world? You can
   find this in changelogs, blogs, bug reports, or perhaps the CVE data.
   Please enter your date in YYYY-MM-DD format.
-announced:
+announced: 2016-05-02
```

Comment:
You're claiming that the vulnerability was announced prior to it being reported by the team. If the vulnerability was made public the team would likely learn of it quickly, double-check this.

Reply:
Ok im not too sure on this actually. The dev confluence page of the vulnerability (https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=65143300) was created on 2016-06-11, but the CVE entry was created or the ID was reserved on 2016-05-02. (https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2016-4430)

Reply:
Just pushed a change for this. The CVE date could have just been a reservation so I found another resource where the vulnerability was announced publicly.